homeassistant/components/transmission/translations/en.json

```diff
@@ -12,6 +12,8 @@
             "user": {
                 "data": {
                     "host": "Host",
+                    "limit": "Limit",
```

Comment:
The limit and order strings should not be in the main flow.

Reply:
@MartinHjelmare could you please give me more details or examples, I don't really get the idea

Reply:
There's no form in the main config flow where we ask the user for the limit and the order. That form exists in the options flow. The strings for the options flow form should be under the `"options"` key in the `strings.json` file.

See eg:
https://github.com/home-assistant/core/blob/4a374f037864aad9330caaf39c48656058acb654/homeassistant/components/airvisual/strings.json#L40-L49